tests/services/order_validation_service/integration_tests.py

```diff
@@ -0,0 +1,15 @@
+from cg.services.order_validation_service.workflows.tomte.models.order import TomteOrder
+from cg.services.order_validation_service.workflows.tomte.validation_service import (
+    TomteValidationService,
+)
+
+
+def test_valid_order(valid_order: TomteOrder, tomte_validation_service: TomteValidationService):
+
+    # GIVEN a valid order
+
+    # WHEN validating the order
+    errors = tomte_validation_service.validate(valid_order.model_dump_json())
+
+    # THEN no errors should be raised
+    assert not errors
```

Comment:
Nice 👍  Maybe we can, at a later stage, add tests for the following
- error in an order field
- error in a case sample field
- error in a sample field
- error in basic data type for field